drivers/i2c/i2c_esp32.c

```diff
@@ -676,6 +678,9 @@ static int IRAM_ATTR i2c_esp32_init(const struct device *dev)
 	I2C_ESP32_FREQUENCY(DT_INST_PROP(idx, clock_frequency))
 
 #define ESP32_I2C_INIT(idx)		\
+					\
+PINCTRL_DT_INST_DEFINE(idx);		\
```

Comment:
nit: please align with other macro content

Reply:
This driver will be addressed again in another PR, the scope of that PR also includes some code refactoring.

If you don't mind, I'd rather address this alignment suggestion in that PR to not cloud pinctrl-related modifications in here, ok?